src/neo/Network/P2P/TaskManager.cs

```diff
@@ -271,6 +272,11 @@ private void RequestTasks(TaskSession session)
             else if (Blockchain.Singleton.HeaderHeight >= session.LastBlockIndex
                     && TimeProvider.Current.UtcNow.ToTimestamp() - PingCoolingOffPeriod >= Blockchain.Singleton.GetBlock(Blockchain.Singleton.CurrentHeaderHash)?.Timestamp)
```

Comment:
```suggestion
                    && TimeProvider.Current.UtcNow.ToTimestamp() - PingCoolingOffPeriod >= Blockchain.Singleton.GetBlock(Blockchain.Singleton.CurrentHeaderHash)?.Timestamp/1000)
```

@nicolegys have found this bug. The unit of `TimeProvider.Current.UtcNow.ToTimestamp` is second, and the unit of  `block.Timestamp` is microsecond.


@shargon The last bug need to fix. 😂

Reply:
![image](https://user-images.githubusercontent.com/43694095/82438293-631b6600-9acb-11ea-9af8-69ee4c7e3bee.png)
😂

Reply:
Aheuahea, makes sense, this bug is sense the MS change. It should be `ToTimestampMS`

Reply:
@Tommo-L, maybe @nicolegys can open this in a new PR for her to commit this separately.

Reply:
@nicolegys what do you prefer?

Reply:
@shargon I think you can fix it in this PR. 
`TimeProvider.Current.UtcNow.ToTimestampMS() - PingCoolingOffPeriod * 1000 >= Blockchain.Singleton.GetBlock(Blockchain.Singleton.CurrentHeaderHash)?.Timestamp` is better for me : )

Reply:
Redefine the `PingCoolingOffPeriod`
```cs
private const int PingCoolingOffPeriod = 60000; // in secconds.
```

Reply:
Please, check it